### ISSUE TITLE
loop hole

### ISSUE THREAD

**Issue by Souravbudke** [AzeemIdrisi/QR-Attendance-System#5]
sir the user can input multiple attendence of him and his friends by just rescanning the qr code again and again

**Comment by AzeemIdrisi** [comment]
Yes, this project is not a complete solution but just a MVP of the idea. It can be improved further in so many ways. The best way I believe is a dedicated app on mobile devices for this purpose which detects duplicate submissions.
